staging/src/k8s.io/component-base/featuregate/feature_gate_test.go

```diff
@@ -420,6 +424,33 @@ func TestFeatureGateSetFromMap(t *testing.T) {
 	}
 }
 
+func TestFeatureGateMetrics(t *testing.T) {
+	// gates for testing
+	featuremetrics.ResetFeatureInfoMetric()
+	const testAlphaGate Feature = "TestAlpha"
+	const testBetaGate Feature = "TestBeta"
+	testedMetrics := []string{"kubernetes_feature_enabled"}
+	expectedOutput := `
+		# HELP kubernetes_feature_enabled [ALPHA] This metric records the data about the stage and enablement of a k8s feature.
+        # TYPE kubernetes_feature_enabled gauge
+        kubernetes_feature_enabled{name="TestAlpha",stage="ALPHA"} 0
+        kubernetes_feature_enabled{name="TestBeta",stage="BETA"} 1
+        kubernetes_feature_enabled{name="AllAlpha",stage="ALPHA"} 0
+        kubernetes_feature_enabled{name="AllBeta",stage="BETA"} 0
+`
+
+	f := NewFeatureGate()
```

Comment:
add two additional featuregate entries to this unit test that would have caught the Default vs Enabled issue
* one with Default true that is explicitly disabled
* one with Default false that is explicitly enabled